File: backend/entidades/Pedidos.cs
namespace backend.entidades
{
    public class Pedidos : Common
    {
        public int Id { get; set; }
        public int IdUsuario { get; set; }
        public int IdEnvio { get; set; }
        public int fechaPedido { get; set; }
        public decimal Cantidad { get; set; }
        public string Estado { get; set; }
    

    }
}

File: frontend/frontend-upc/src/app/entidades/pedidos.ts
export class Pedidos{
  id: number;
  idUsuario: number;
  idEnvio: number;
  fechaPedido: Date;
  estado: string;
  cantidad: string;

  constructor(
    id: number,
    idUsuario: number,
    idEnvio: number,
    fechaPedido: Date,
    estado: string,
    cantidad: string,
  ) {

    this.id = id;
    this.idUsuario = idUsuario;
    this.idEnvio = idEnvio;
    this.fechaPedido = fechaPedido;
    this.estado = estado;
    this.cantidad = cantidad;
  }
}